package.json

```diff
@@ -80,6 +80,7 @@
     "test:only": "if grep -r --exclude-dir=node_modules --exclude-dir=.git --color 'test\\.only' ; then exit 1; fi",
     "test:bail": "npm run test:raw | tap-arc --bail",
     "test": "npm run test:raw | tap-arc && npm run test:only",
+    "lint": "eslint .",
```

Comment:
Good, I've been missing this - I caught an "unused var" the other day, or maybe undefined var.
Anyway, I think this should be part of `npm test` script

Reply:
added to npm test